contracts/USDPImplementationV3.sol

```diff
@@ -17,27 +16,21 @@ import "./zeppelin/SafeMath.sol";
  * Any call to transfer against this contract should fail
  * with insufficient funds since no tokens will be issued there.
  */
-contract USDPImplementationV3 {
-
-    /**
-     * MATH
-     */
-
-    using SafeMath for uint256;
+contract USDPImplementationV3 is PaxosBaseAbstract{
 
     /**
      * DATA
      */
 
     // INITIALIZATION DATA
-    bool private initialized = false;
+    bool private initialized;
 
     // ERC20 BASIC DATA
     mapping(address => uint256) internal balances;
     uint256 internal totalSupply_;
-    string public constant name = "Pax Dollar"; // solium-disable-line
-    string public constant symbol = "USDP"; // solium-disable-line uppercase
-    uint8 public constant decimals = 18; // solium-disable-line uppercase
+    string public constant name = "Pax Dollar"; // solhint-disable-line const-name-snakecase
+    string public constant symbol = "USDP"; // solhint-disable-line const-name-snakecase
+    uint8 public constant decimals = 6; // solhint-disable-line const-name-snakecase
```

Comment:
This shouldn't have changed!

Reply:
True. I caught it yesterday in testing and should have added a comment that this is an issue.